COPYRIGHT.md

```
@@ -1,4 +1,4 @@
-Copyright © 2025 The Stable Order <br />
+Copyright © 2026 The Stable Order <br />
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)